playbooks/update_files.yml

```diff
@@ -197,9 +217,9 @@
                 echo Existing PR "$prnum"
                 echo {{ github_url_prefix ~ github_org ~ '/' ~ inventory_hostname ~ '/pull/' }}$prnum
               fi
-      always:
-        - name: Clean up temp working dir
-          file:
-            path: "{{ __working_dir.path }}"
-            state: absent
-          when: test_dir is not defined
```

Comment:
If you run with e.g. `-e test_dir=/var/tmp/lsr` then it will keep that directory and you don't have to comment this out.

Reply:
Yeah sorry I commited everything by mistake